README.md

```diff
@@ -175,5 +168,23 @@ book.add_author(author_name)
 
 ```
 
+### Get Author's age
+
+```python
+    def get_author_age(self):
```

Comment:
You were supposed to put sample code for calling your method, not the method definition.